src/main/resources/templates/db_scripts/schema.json

```diff
@@ -612,6 +634,13 @@
           "fieldName": "exportJobId",
           "caseSensitive": false
         }
+      ],
+      "ginIndex": [
+        {
+          "fieldName": "exportedPoLineIds",
+          "caseSensitive": false,
+          "removeAccents": true
+        }
       ]
```

Comment:
Additional WARN logs for Create Order, Create PO Line, Get Order, Get PO Line, Open Order, Receive Title will be covered by correcting UI queries(3 places to improve) in scope of this story:
https://issues.folio.org/browse/UIOR-1181